Sentence casing (#1037)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md

---
title: "Configuring notifications"

> description: "You receive Email or Slack alerts for job run status when you set up notification in dbt Cloud."

Author: ```suggestion
description: "Set up notifications in dbt Cloud to receive Email or Slack alerts for job run status."
```
